samples/nrf9160/coap_client/README.rst

```diff
@@ -32,13 +45,21 @@ Testing
 
 After programming the sample and all prerequisites to the board, test it by performing the following steps:
 
-1. Connect the USB cable and power on or reset your nRF9160 DK.
-#. Open a terminal emulator and observe that the kit prints the following information::
+1. Connect your nRF9160 DK to the PC using USB cable and power on or reset your nRF9160 DK.
+#. Open a terminal emulator and observe that the following information is displayed::
 
        The nRF CoAP client sample started
-#. Observe that the kit sends periodic CoAP GET requests after it gets LTE connection.
-   The kit sends requests to the configured server (``CONFIG_COAP_SERVER_HOSTNAME``) for a configured resource (``CONFIG_COAP_RESOURCE``).
-#. Observe that the kit either prints the reponse data received from the server or indicates a time-out.
+#. Observe that the following AT commands are displayed on the terminal emulator:
```

Comment:
Should we have the following documentation for the AT commands that are generated after the sample starts?

Reply:
I don't think we should, for instance I'm not seeing them (only CEREG notifications)? The truth is that we see them as a side-effect of some other module, they will once corrected (I haven't investigated who prints them though).

Reply:
Removed the documentation for AT commands.